docs/content/preview/migrate/install-yb-voyager.md

```diff
@@ -60,41 +71,68 @@ The export directory has the following sub-directories and files:
-
-  ```sh
-  ./install-yb-voyager
-  ```
-
-  It is safe to execute the script multiple times. If the script fails, check the `/tmp/install-yb-voyager.log` file.
-
-- The script generates a `.yb-voyager.rc` file in the home directory. Source the file to ensure that the environment variables are set using the following command:
-
-  ```sh
-  source ~/.yb-voyager.rc
-  ```
-
-- Check that yb-voyager is installed using the following command:
 
-  ```sh
-  yb-voyager --help
-  ```
+Install yb-voyager on a machine which satisfies the [Prerequisites](#prerequisites) using one of the following options:
+
+<ul class="nav nav-tabs nav-tabs-yb">
+  <li>
+    <a href="#linux" class="nav-link active" id="linux-tab" data-toggle="tab" role="tab" aria-controls="linux" aria-selected="true">
+      <i class="fa-brands fa-redhat" aria-hidden="true"></i>
+      RHEL
+    </a>
+  </li>
+  <li>
+    <a href="#ubuntu" class="nav-link" id="ubuntu-tab" data-toggle="tab" role="tab" aria-controls="ubuntu" aria-selected="true">
+      <i class="fa-brands fa-ubuntu" aria-hidden="true"></i>
+      Ubuntu
+    </a>
+  </li>
+    <li >
+    <a href="#macos" class="nav-link" id="macos-tab" data-toggle="tab" role="tab" aria-controls="macos" aria-selected="true">
+      <i class="fa-brands fa-apple" aria-hidden="true"></i>
+      macOS
+    </a>
+  </li>
+  <li>
+    <a href="#airgapped" class="nav-link" id="airgapped-tab" data-toggle="tab" role="tab" aria-controls="airgapped" aria-selected="true">
+      <i class="fa-solid fa-link-slash" aria-hidden="true"></i>
+      Airgapped
+    </a>
+  </li>
+  <li>
+    <a href="#docker" class="nav-link" id="docker-tab" data-toggle="tab" role="tab" aria-controls="docker" aria-selected="true">
+      <i class="fa-brands fa-docker" aria-hidden="true"></i>
+      Docker
+    </a>
+  </li>
+  <li>
+    <a href="#github" class="nav-link" id="github-tab" data-toggle="tab" role="tab" aria-controls="github" aria-selected="true">
+      <i class="fab fa-github" aria-hidden="true"></i>
+      Source
+    </a>
+  </li>
+</ul>
+
+<div class="tab-content">
+  <div id="linux" class="tab-pane fade show active" role="tabpanel" aria-labelledby="linux-tab">
+{{% readfile "./linux.md" %}}
+  </div>
+  <div id="ubuntu" class="tab-pane fade" role="tabpanel" aria-labelledby="ubuntu-tab">
+{{% readfile "./ubuntu.md" %}}
+  </div>
+  <div id="macos" class="tab-pane fade" role="tabpanel" aria-labelledby="macos-tab">
+{{% readfile "./macos.md" %}}
+  </div>
+  <div id="airgapped" class="tab-pane fade" role="tabpanel" aria-labelledby="airgapped-tab">
+{{% readfile "./airgapped.md" %}}
+  </div>
+  <div id="docker" class="tab-pane fade" role="tabpanel" aria-labelledby="docker-tab">
+{{% readfile "./docker.md" %}}
+  </div>
+  <div id="github" class="tab-pane fade" role="tabpanel" aria-labelledby="github-tab">
+{{% readfile "./github.md" %}}
+  </div>
+</div>
 
 To learn more about yb-voyager, refer to [YugabyteDB Voyager CLI](../yb-voyager-cli/).
```

Comment:
We should drop this. The page already has a `Next step` directing users to `Migration steps`.